crates/starknet-server/tests/test_call.rs

```diff
@@ -13,6 +13,7 @@ mod call {
     use crate::common::devnet::BackgroundDevnet;
 
     #[tokio::test]
+    /// This test doesn't rely on devnet.get_balance because it's not supposed to call ERC20
```

Comment:
I don't get this comment. Shouldn't this be "This test doesn't rely on balanceOf call"?

Reply:
I only added this comment to explain why I'm not using the `get_balance` util method, because someone might think it could be used here.

Let's look at what this test case really does. As the name says, it's `calling_method_of_undeployed_contract`. Whoever wrote this test case (looks like it was me), decided to test how devnet behaves on calling an undeployed contract by trying to simulate the calling of ERC20, but with an invalid address (i.e. an undeployed address, i.e. an address where nothing has been deployed).

If you look at the tests in this file (they are all testing `starknet_call`):
- the first one is calling, but with a wrong address
- the second one is calling, but with a wrong method (non-existent method selector)
- the third one is calling the correct address with a correct method (happy path)

Reply:
ok got it now